hw02-generics/src/main/java/homework/CustomerReverseOrder.java

```diff
@@ -1,14 +1,16 @@
 package homework;
 
-@SuppressWarnings({"java:S1186", "java:S1135", "java:S1172"}) // при выполнении ДЗ эту аннотацию надо удалить
+import java.util.LinkedList;
+
 public class CustomerReverseOrder {
 
-    // todo: 2. надо реализовать методы этого класса
-    // надо подобрать подходящую структуру данных, тогда решение будет в "две строчки"
+    private final LinkedList<Customer> customers = new LinkedList<>();
```

Comment:
Тип лучше объявлять интерфейсом. Причем минимально необходимым.
```suggestion
    private final Deque<Customer> customers = new LinkedList<>();
```